### PR TITLE
MASSIVE overhaul to history command.

### PR DESCRIPTION
**Pull request by capslock321**
Allowed choosing of channel and gives the user a ✅ that the command is processing and it is not broken.
Yeah so idk if u gonna accept this...

### CODE REVIEW

File: cogs/checks.py [hunk 1]
```diff
@@ -221,17 +222,22 @@ def reaction_check(m):
...
         counter = 0
-        async for message in ctx.channel.history(limit=limit):
+        embed = discord.Embed(title = "Processing Request... (This will take a while) ")
+        await ctx.send(embed = embed)
+        async for message in channel.history(limit=limit):
             if message.author == self.bot.user:
                 counter += 1
                 if counter == limit:
-                    await ctx.send('Your channel History is to Large!')
+                    await ctx.send("This channel's history is too large!")
```
Comment (Seniatical) [inline cogs/checks.py:236]: Could've added a break here or else it would continute on and on

**Comment by capslock321** [comment]
sure ill change
edit: resolved

**Comment by Seniatical** [comment]
Also you can massively improve the version

**Comment by Seniatical** [comment]
Fix the command and make sure any bugs arent present

**Comment by capslock321** [comment]
will do sir

**Pull request merged**